6_temp_coop_fetch.yml

```diff
@@ -17,46 +17,36 @@ targets:
 
   6_temp_coop_fetch:
     depends:
-      - 6_temp_coop_fetch/downloads/long_term_ntl.csv.ind
-      # THIS IS FAILING TO DOWNLOAD - 6_temp_coop_fetch/downloads/mendota_daily_buoy.csv.ind
-      # THIS IS FAILING TO DOWNLOAD - - 6_temp_coop_fetch/downloads/mendota_temps_long.csv.ind
+
       - 6_temp_coop_fetch/log/6_temp_coop_fetch_tasks.ind
 
   # -- download LTER files and push to GD -- #
   coop_file_upload_location:
     command: as_id(I('1dutCiFEOoRObXLn6n3BRKhZDDAijK65Q'))
 
-  6_temp_coop_fetch/downloads/long_term_ntl.csv.ind:
-    command: download_and_indicate(in_ind = target_name,
-             url = I("https://lter.limnology.wisc.edu/sites/default/files/data/ntl29_v5.csv"),
```

Comment:
It was originally going into that `in` automatically too, since in the `download_and_indicate()` function:
```r
googledrive::drive_upload(media = local_file, path = push_location)
```
Where `push_location` is the google drive location for the `in` folder. So this was going into `downloads` _and_ into `in`. `in` is where it became part of the coop parsers and I think the `downloads` folder is ignored by this pipeline. 

> Is the download URL still being documented somewhere?
yes, I added an explainer file for this target.